contributors.txt

```diff
@@ -102,4 +102,5 @@ YYYY/MM/DD, github id, Full name, email
 2019/10/13, bansan85, Le Garrec Vincent, legarrec.vincent{at]gmail.com
 2019/10/14, FabMan08, Fabrizio Mancin, fabman08(at)gmail.com
 2019/10/15, manishkungwani, Manish Kungwani, manishkungwani{at]yahoo.com
-2019/10/18, lhiginbotham, Logan Higinbotham, logan.higinbotham(at)gmail.com
\ No newline at end of file
+2019/10/18, lhiginbotham, Logan Higinbotham, logan.higinbotham(at)gmail.com
+2019/10/18, idealist1508, none, none(at)none.de
```

Comment:
❗️  You must provide your real name and your real email

Please re-read the document

Reply:
I am sorry, but i don't wont to publish my real name and real email publicly. This is not acceptable for me.

Reply:
You could add a separate GitHub account (which I believe is OK from GitHub's point of view), with an email address separate from your normal. But it should be a real monitored account, not a fake one.